src/views/registration/attendees/fields/radio.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * @version TBD
+ *
+ */
+$field    = $this->get( 'field' );
+$required = isset( $field->required ) && 'on' === $field->required ? true : false;
+$field    = (array) $field;
+
+$options = null;
+
+if ( isset( $field['extra'] ) && ! empty( $field['extra']['options'] ) ) {
```

Comment:
lets use `Tribe__Utils__Array::get( array( 'extra', 'options' ) )` to avoid conditional